src/test/run-pass-fulldeps/proc-macro/attr-args.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// aux-build:attr-args.rs
+
+#![allow(warnings)]
+#![feature(proc_macro)]
+
+#[macro_use]
+extern crate attr_args;
```

Comment:
If `#![feature(proc_macro)]` implied `#![feature(use_extern_macros)]`, then we would prefer
```rust
extern crate attr_args;
use attr_args::attr_with_args;
```
here.